tutorials/navigation/navigation_using_agent_avoidance.rst

```diff
@@ -0,0 +1,79 @@
+.. _doc_navigation_using_agent_avoidance:
+
+Using Agent Avoidance
+=====================
+
+.. warning::
+
+    This chapter is a stub for linking and unfinished.
```

Comment:
See my note about this warning in https://github.com/godotengine/godot-docs/pull/6048

Reply:
removed